src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -621,6 +621,7 @@
     <Reference Include="System.Diagnostics.Tools" />
     <Reference Include="System.Diagnostics.Tracing" />
     <Reference Include="System.IO.Compression" />
+    <Reference Include="System.Linq" />
```

Comment:
Why is this needed?  If we're depending on LINQ from SocketsHttpHandler, we're likely doing something wrong.

Reply:
It's not needed, I forget to delete it. Will remove.